src/test/java/gustavohuanca/code/typeCall/CallStatus.java

```diff
@@ -0,0 +1,18 @@
+package gustavohuanca.code.typeCall;
+
+import static io.restassured.RestAssured.given;
+
+public class CallStatus implements IRequestCall{
+    @Override
+    public String answerToCall(String pathEndPoint) {
+        return responseCallStatus(pathEndPoint)+"";
+    }
+
+    private int responseCallStatus(String pathEndPoint){
+        return given()
+                .when()
+                .get("http://zippopotam.us/us/90210")
```

Comment:
the url and endpoint should not be here. should be in the setting parameter. why are not using it?